themes/bootstrap3/templates/footer.phtml

```diff
@@ -28,6 +28,9 @@
           <li><a href="<?=$this->url('help-home')?>?topic=search&amp;_=<?=time() ?>" data-lightbox class="help-link"><?=$this->transEsc('Search Tips')?></a></li>
           <li><a href="<?=$this->url('content-page', ['page' => 'asklibrary']) ?>"><?=$this->transEsc('Ask a Librarian')?></a></li>
           <li><a href="<?=$this->url('content-page', ['page' => 'faq']) ?>"><?=$this->transEsc('FAQs')?></a></li>
+          <?php if ($this->plugin('cookieConsent')->isEnabled()): ?>
```

Comment:
That's to get the plugin without calling __invoke. I suppose the other option would be to change __invoke, but I'm not sure which one is better.

Reply:
Ahh, I see! If we need to avoid calling __invoke, it's that a sign that the invoke method is doing too much work? (The helper code is another piece I haven't reviewed yet, so again, my apologies if the answer is obvious... I'll find out in a couple of days when I have more time!)

Reply:
Indeed. I flip-flopped a bit about it but left it as is for now. The primary function of outputting the code needed for the cookie consent component is now done via __invoke the same way it's done with most simple helpers, but this can obviously be changed.

Reply:
Makes sense. Either way, it makes some code more verbose and other code less verbose. I think having a render() method on the helper instead of rendering with __invoke might be slightly more clear, though -- generally if a helper has multiple callable methods, we just use the default __invoke (which just returns the object itself).

Reply:
I failed to mention that I added the render() method instead of __invoke doing it. Now that the plugin has several more methods available, it makes even more sense this way.